extide/gradle/src/org/netbeans/modules/gradle/loaders/ExtensionPropertiesExtractor.java

```diff
@@ -170,7 +171,7 @@ public Collection<String> keys(BuildPropertiesSupport.Property container) {
             if (c == null) {
                 return null;
             }
-            return Arrays.asList(c.split(";;")).stream().map(s -> s.replaceAll("\\;", ";")).collect(Collectors.toList());
+            return Arrays.asList(c.split(";;")).stream().map(s -> s.replaceAll("\\\\;", ";")).collect(Collectors.toList());
```

Comment:
`s.replace(..)` wouldn't need the double escape I believe - its also cheaper since it doesn't do regex.

Reply:
Apologies, completely forgot about this - fixed in 8f0cfa2c1683